src/Data/Data.php

```diff
@@ -41,40 +41,40 @@ class Data
      * @var array
      */
     public static $handlers = [
-        'json' => 'Kirby\Data\Json',
-        'php'  => 'Kirby\Data\PHP',
-        'txt'  => 'Kirby\Data\Txt',
-        'yaml' => 'Kirby\Data\Yaml',
+        'json' => Json::class,
```

Comment:
There was a reason for the strings. They don't trigger the autoloader. With the class constant the autoloader will automatically load the classes even if they are not used afterwards.

Reply:
Oh wow, I didn't know that. That's definitely stupid then. I have reverted this change.